sdks/android/jni/runtime-bootstrap.c

```diff
@@ -417,6 +461,44 @@ Java_org_mono_android_AndroidRunner_runTests (JNIEnv* env, jobject thiz, jstring
 	}
 
 	mono_runtime_invoke (run_tests_method, NULL, NULL, NULL);
+#elif MONO_DEBUGGER_TESTS
```

Comment:
there is quite some code duplication from code above

Reply:
Good test code, in my opinion, is easy to read and easy to edit while debugging. There's really just duplication between lines 468 and 495, and I'd say it's arguable what is accomplished by making the name of the class and method an argument to a helper method. It's likely that one of these two will need to be changed in the near future, and three identical function calls with error handling is hardly a lot of duplication.

Reply:
:+1: